extension/intents/clipboard/clipboard.toml

```diff
@@ -21,12 +21,15 @@ test = true
 [clipboard.copyLink]
 description = "Copies the link of the current tab"
 match = """
-  copy (the |) (this |) (tab's |) (link | url) (of this tab |) (to clipboard |)
+  copy (the |) (this |) (tab's |) (link{s} | url) (of this tab |) (to clipboard |)
 """
 
 [[clipboard.copyLink.example]]
 phrase = "Copy link"
 
+[[clipboard.copyLink.example]]
+phrase = "Copy links"
```

Comment:
We don't need the user to see both examples ("copy link" and "copy links"), so you could add `test = true` to this example.